collection-scripts/gather

```diff
@@ -25,13 +31,18 @@ resources+=(storageclasses persistentvolumes volumeattachments)
 resources+=(clusternetworks hostsubnets)
 
 # Autoscaler Resources
-resources+=(clusterautoscaler machineautoscaler)
+machineresources+=(clusterautoscaler machineautoscaler)
 
 # Run the Collection of Resources using must-gather
 for resource in ${resources[@]}; do
     /usr/bin/openshift-must-gather inspect ${resource}
 done
 
+# Run the Collection of MachineAPIResources using must-gather
```

Comment:
Why collect this separate? Why not just included this in **resources**?

Reply:
According to https://docs.openshift.com/container-platform/4.1/machine_management/applying-autoscaling.html the `clusterautoscaler` is something you would find in the `default` namespace while the `machineautoscaler` is something you find out of the `openshift-machine-api` namespace. 

Should we just do some loop or update the loop above to get things from all namespaces?

Reply:
Let's consider https://github.com/openshift/must-gather/pull/127/files over this PR.